src/SUMMARY.md

```diff
@@ -2,22 +2,23 @@
 
 [Introduction](README.md)
 
+# Getting started
+
+- [Quick start](quick_start.md)
+
 # User manual
 
-- [Getting started](getting_started.md)
+- [Security controls](controls.md)
+- [Configuration file format](helpdoc/configuration.md)
+- [Checktype catalog file format](helpdoc/checktypes.md)
+- [Metrics collection](helpdoc/metrics.md)
+- [Environment variables](helpdoc/environment.md)
 
-# Commands
+# Command Line Interface
```

Comment:
If you are using lowercase in the other sections, wouldn't it be `Command line interface`?